templates/includes/footer.html

```diff
@@ -5,16 +5,17 @@
         <h3>Contact Info</h3>
         <div class="content">
           <p>
-            OSU Open Source Lab<br />
-            Kerr Admin B211<br />
-            Corvallis, OR 97331<br />
-            General Inquiries:<br /><span style="color: #C34500;"><a href="mailto:info@osuosl.org">info@osuosl.org</a></span><br />
-            Support for Project Infrastructure<br /><span style="color: #C34500;"><a href="mailto:support@osuosl.org">support@osuosl.org</a></span><br />
-            Questions about Donations:<br /><span style="color: #C34500;"><a href="mailto:donations@osuosl.org">donations@osuosl.org</a></span>
+            {{ SITENAME }}-- {{ NAME_ABBREV }}<br />
```

Comment:
I think this is a little awkward - using the current CASS contact info, the name-- abbreviation line wraps, and looks like this:
```
Center for Applied 
Systems and Software-- 
CASS
209 & 224 Milne 
Computer Center
1800 SW Campus Way 
Corvallis, OR 97331
info@CASS.oregonstate.edu
Phone: 541-737-2060 
```

@kelnera what do you think, should we re-arrnage the abbreviation and/or remove the --?